apps/docs/pages/guides/integrations/prisma.mdx

```diff
@@ -153,7 +143,7 @@ Update your `.env` file with the following changes:
 1. Rename the `DATABASE_URL` environment variable to `DIRECT_URL`
 1. Create a `DATABASE_URL` environment variable and paste in the new connection string from the dashboard as its value
 
-It is recommended to minimize the number of concurrent connections by setting the `connection_limit` to `1`. You can set connection limit by appending `?connection_limit=1` to your connection string
+Append the `?pgbouncer=true` flag to the `DATABASE_URL` variable. It is recommended to minimize the number of concurrent connections by setting the `connection_limit` to `1`. You can set connection limit by appending `?connection_limit=1` to your connection string. 
```

Comment:
The `connection_limit=1` recommendation is terrible an misleading and will ensure all queries of a request are run sequentially. This should just be removed.

(If this is for some serverless use case, then it should at least be split out and mentioned separately with a proper explanation)

Reply:
I've moved up "It is recommended to minimize the number of concurrent connections by setting the `connection_limit` to `1`." to the first paragraph. Let me know if it reads better and any further adjustments.

Reply:
I've removed any information on using the `connection_limit`. Let me know if it reads better.